### ISSUE TITLE
Use markdown instead of MS Word

### ISSUE THREAD

**Issue by Samyak2** [varunp2k/OverTheWire-Solutions#7]
## The Issue

The solutions are saved to a .docx file. Which has multiple issues:
 - It cannot be viewed through GitHub, since it's a binary data.
 - MS Word is a proprietary closed source software, which goes against what Open Source stands for.
 - Changes cannot be tracked through GitHub or Git.

## The solution

 - Use a .md file to write the solutions formatted using markdown.
 - Upload images to either GitHub or a third-party image hosting service like imgur.
 - Write the commands and output using backquotes instead of uploading images of them. (just a suggestion)

## Why Markdown?

 - Markdown is easier to format. Code or commands can be inserted in a simple and good looking way.
 - Markdown is basically plain text, hence it takes lesser space.
 - It's portable - does not require a heavy software to open it. It can be edited using any text editor.
 - GitHub renders markdown formatted text when opened.

**Comment by Gituser143** [comment]
Samyak, that's a brilliant point made. Can you take up the responsibility to migrate whatever we've got from word to markdown though? It would be really helpful...

**Comment by Samyak2** [comment]
Yes, I would be happy to help migrate it. Give me a few days, I'll send a PR.

**Comment by varunp2k** [comment]
I will add this to  Hacktober fest by tonight

**Comment by Samyak2** [comment]
I found a mistake in the doc. The image for Level 6 to 7 shows the problem page of Level 5 to 6.
![image](https://user-images.githubusercontent.com/34161949/66131815-b3fc4e00-e611-11e9-8891-6dccc65a148d.png)

**Comment by Samyak2** [comment]
I have converted solutions upto Level 12 (upto the level I have completed :p). Check the [md file here](https://github.com/Samyak2/OverTheWire-Solutions/blob/samyak-patch-md-1/CTF.md) and let me know if it looks good. I will continue converting them.

**Comment by Gituser143** [comment]
@Samyak2, seems fine. Send a pull request? I'll add the Level 6 to 7 thing as an issue.
